### PR TITLE
Copyedits and markdown formatting per Astro-723

### PR DESCRIPTION
**Pull request by sknep**
Tickets: 
https://rocketcom.atlassian.net/browse/ASTRO-723 
https://rocketcom.atlassian.net/browse/ASTRO-718
https://rocketcom.atlassian.net/browse/ASTRO-710
https://rocketcom.atlassian.net/browse/ASTRO-711
Revised Content: 
https://rocketcom.atlassian.net/wiki/spaces/AS11/pages/679674707/Getting+Started+with+Astro
Review changes here: 
https://deploy-preview-88--astrouxds.netlify.app/getting-started/readme/

### CODE REVIEW

**Comment by sknep** [comment]
It sounds like there are more copyedits coming, hold please

**Comment by sknep** [comment]
I found some other issues on https://deploy-preview-88--astrouxds.netlify.app/getting-started/developers/ which references some changes to our docs and installation that haven't actually been deployed or published yet. It might best to remove this and leave a note about future updates coming soon, but that makes this page largely unhelpful until the content on astro-components' release/v5 branch lands

**Comment by dmcalester** [comment]
This is failing in deploy, but not sure why. It doesn’t seem any of the changes to this PR would cause the issue. Are you able to see the console/build log https://app.netlify.com/sites/astrouxds/deploys/5f765a5e7b9bbc0154bcfdc2 ?

**Comment by sknep** [comment]
OH its because its a draft!

**Comment by sknep** [comment]
>>> 5:39:15 PM: [build:eleventy] Writing _site/sitemap.xml from ./_content/sitemap.njk.
5:39:15 PM: [build:eleventy] Writing _site/index.html from ./_content/index.md.
5:39:15 PM: [build:eleventy] Error writing templates: (more in DEBUG output)
5:39:15 PM: [build:eleventy] > Having trouble writing template: _site/search/index.html
5:39:15 PM: [build:eleventy]
5:39:15 PM: [build:eleventy] `TemplateWriterWriteError` was thrown
5:39:15 PM: [build:eleventy] > (_includes/search.template.njk)
5:39:15 PM: [build:eleventy]   Template render error: (/opt/build/repo/_includes/_nav.njk) [Line 7, Column 33]
5:39:15 PM: [build:eleventy]   Error: Unable to call `data["url"]["split"]`, which is undefined or falsey

**Comment by sknep** [comment]
> This is failing in deploy, but not sure why. It doesn’t seem any of the changes to this PR would cause the issue. Are you able to see the console/build log https://app.netlify.com/sites/astrouxds/deploys/5f765a5e7b9bbc0154bcfdc2 ?

Sorry Duncan, I broke the build :) Fixed now!

File: _content/_data/navigation.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	},
 	{
 		"label": "Getting Started",
+		"url": "/getting-started",
```
Comment (sknep) [inline _content/_data/navigation.json]: It surprised me that we needed a "url" passed in for nav when that page isn't actually used to build a link. Turns out it's needed as a url segment for the sake of sorting the children in the nav.

**Comment by cskerbo** [comment]
> I found some other issues on https://deploy-preview-88--astrouxds.netlify.app/getting-started/developers/ which references some changes to our docs and installation that haven't actually been deployed or published yet. It might best to remove this and leave a note about future updates coming soon, but that makes this page largely unhelpful until the content on astro-components' release/v5 branch lands

**Comment by cskerbo** [comment]
> ### Getting Started  
> You can import Astro UXDS Web Components to an existing application using npm and litElement. Detailed instructions for using Astro components are available in the project’s README.  
>   
> If you’re considering a non-Web Component Astro implementation, <strike>directions for including Astro CSS and other static assets are available here. For those developing offline applications, be sure to review the special considerations for offline Astro assets. </strike> directions for this process will be available soon.

Under the getting started section for Development, we need to make the change above. We can create a separate ticket to ensure this is changed back once the v5 branch is released.

**Review by cskerbo — CHANGES_REQUESTED** [review]
> **Getting Started**
> You can import Astro UXDS Web Components to an existing application using npm and litElement. Detailed instructions for using Astro components are available in the project’s README.
> 
> If you’re considering a non-Web Component Astro implementation, <strike>directions for including Astro CSS and other static assets are available here. For those developing offline applications, be sure to review the special considerations for offline Astro assets.</strike> directions for this process will be available soon.

Under the getting started section for Development, we need to make the change above. We can create a separate ticket to ensure this is changed back once the v5 branch is released.

**Review by cskerbo — APPROVED** [review]

**Comment by sknep** [comment]
@im-cr do you wanna take a look at this branch and see where it stands compared to your work before i merge it in?

**Comment by sknep** [comment]
@dmcalester please approve when you get a moment

**Comment by dmcalester** [comment]
> These examples use a previous release of Astro, and are not intended as boilerplates for building new applications. See below for easy ways to get started building Astro web applications.

:D

File: _content/getting-started/_readme.md [hunk 2]
```diff
@@ -8,43 +8,36 @@ title: "Getting Started"
...
+**Development Assets**
+Astro includes standard HTML and CSS sample code as a starting point for development. Astro's UI Components are provided as Web Components, a platform-neutral technology.
 
-Learn more about designing with Astro development assets, and why we chose to build web components with our guide to getting started [For Developers](/getting-started/developers).
+Learn more about designing with Astro development assets, and why the Astro team chose to build web components with our guide to getting started [For Developers](/getting-started/developers).
 
 ### Support
 
-Supported browser
+**Supported browsers**
```
Comment (dmcalester) [inline _content/getting-started/_readme.md:38]: Title case

File: _content/getting-started/_readme.md [hunk 3]
```diff
@@ -56,21 +49,21 @@ Current version: Astro 4.5
...
+> 
+> RECIPIENT AGREES TO WAIVE ANY AND ALL CLAIMS AGAINST THE UNITED STATES GOVERNMENT, ITS CONTRACTORS AND SUBCONTRACTORS, AS WELL AS ANY PRIOR RECIPIENT. IF RECIPIENT'S USE OF THE SOFTWARE RESULTS IN ANY LIABILITIES, DEMANDS, DAMAGES, EXPENSES OR LOSSES ARISING FROM SUCH USE, INCLUDING ANY DAMAGES FROM PRODUCTS BASED ON, OR RESULTING FROM, RECIPIENT'S USE OF THE SOFTWARE, RECIPIENT SHALL INDEMNIFY AND HOLD HARMLESS THE UNITED STATES GOVERNMENT, ITS CONTRACTORS AND SUBCONTRACTORS, AS WELL AS ANY PRIOR RECIPIENT, TO THE EXTENT PERMITTED BY LAW. RECIPIENT'S SOLE REMEDY FOR ANY SUCH MATTER SHALL BE THE IMMEDIATE, UNILATERAL TERMINATION OF THIS AGREEMENT.
 
 
 ### Offline access
```
Comment (dmcalester) [inline _content/getting-started/_readme.md:69]: Title case

**Review by dmcalester — APPROVED** [review]
Approved with two minor case changes to titles.

**Pull request merged**